gluster_georep_tools/setup/cli.py

```diff
@@ -173,7 +173,7 @@ def execute(cmd, success_msg="", failure_msg="", exitcode=-1):
     if p.returncode == 0:
         if success_msg:
             output_ok(success_msg)
-        return out
+        return out.decode('utf-8')
```

Comment:
adding `universal_newlines=True` argument to Popen(Line 171) to fix this issue.

Reply:
Addressed